industry_restaurant/data/ir_attachment_pre.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="ir_attachment_602" model="ir.attachment">
+        <field name="name">unsplash_y3aP9oo9Pjc_.jpg</field>
+        <field name="datas" type="base64" file="industry_restaurant/static/src/binary/ir_attachment/602-unsplash_y3aP9oo9Pjc_.jpg"/>
+    </record>
+    <record id="ir_attachment_611" model="ir.attachment">
```

Comment:
Do you need to define the ir.attachment, as it is only used defining the complete path?